src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const result = [];
+
+  for (const act in actions) {
+    const type = actions[act].type;
+
+    if (type === 'addProperties') {
```

Comment:
According to the checklist it's better to use switch statement 🙂

Reply:
fixed it, thanks)